pkg/compiler/compilation_document.go

```diff
@@ -66,6 +90,25 @@ func (doc *CompilationDocument) OutputTo(dest string) error {
 	return nil
 }
 
+func postCompileHook(dir string, file core.File, hook string, log *zap.SugaredLogger) error {
+	hookSegments := strings.Split(hook, " ")
+	if len(hookSegments) == 0 {
+		return errors.New(`empty formatter command`)
+	}
+	var args []string
+	for _, arg := range hookSegments[1:] {
+		if arg == "{}" {
+			arg = file.Path()
+		}
+		args = append(args, arg)
+	}
+
+	cmd := exec.Command(hookSegments[0], args...)
+	log.Infof(`running post-output hook: %s`, strings.Join(cmd.Args, " "))
```

Comment:
nit: edge-case, but might want to quote all the args if the intention is that the command is copy/paste-able for reproducing. This also might be more of a Debug log if it can run over user files (see general comment `(1)`).

Reply:
That wasn't my intention, but it's a good idea.